src/app/wallets/get-all-wallets.ts

```diff
@@ -0,0 +1,6 @@
+import { WalletsRepository } from "@services/mongoose"
+
+export const getAllWallets = async (walletCurrency: WalletCurrency) => {
+  const wallets = await WalletsRepository().listAll(walletCurrency)
+  return wallets
```

Comment:
```suggestion
  return WalletsRepository().listAll(walletCurrency)
```

this should include pagination else we will have performance issues